dashboard/templates/builds_history.html

```diff
@@ -40,6 +40,7 @@ <h1>Builds history for {{ buildsets.pipeline }}</h1>
         {% for b in buildsets.page %}
         <div class="row hover-row build-table clickable" data-toggle="collapse"
              data-target="#collapse-{{ b.id }}">
+            <div class="dropdown-toggle"></div>
```

Comment:
When you move this, you should also move or remove ``<div></div>`` from line 38 - it's only a placeholder for arrow in table head, but now I see that remove this changes nothing, so probably it's to remove.